src/utils/database.ts

```diff
@@ -0,0 +1,40 @@
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+export class Database {
+  defaultFilePath: string;
+
+  constructor() {
+    this.defaultFilePath = 'src/utils/data.json';
+  }
+
+  async hasGameFile(filePathOverride?: string): Promise<boolean> {
+    return existsSync(filePathOverride || this.defaultFilePath);
+  }
+
+  async load(filePathOverride?: string): Promise<object> {
+    try {
+      const data = await readFileSync(
+        filePathOverride || this.defaultFilePath,
+        'utf-8'
+      );
+      return JSON.parse(data);
```

Comment:
`JSON.parse` returns `any` which is not an object. I also updated the [linting PR](https://github.com/normal-devs/klicker-knight/pull/6) to catch casting `any` to something else. This is technically valid typescript but I don't think we want this behavior.

Reply:
I updated this to `unknown`. I'm not sure if this is any better...

Reply:
I think we want a different layer to verify the data structure, so `unknown` is a good option. It prevents the next layer from operating on the data before narrowing the type of it. See [this stack overflow post](https://stackoverflow.com/a/57356103)